lte/protos/session_forwarder.proto

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package magma.lte;
+
+import "orc8r/protos/common.proto";
+import "feg/protos/s8_proxy.proto";
+
+option go_package = "magma/lte/cloud/go/protos";
+
+service SessionForwarder {
+  rpc ReportSessionStats(ReportSessionStatsRequest) returns (orc8r.Void) {}
```

Comment:
I think we need to create new message `ReportSessionsStatsRequest` which contains array of `ReportSessionStatsRequest`